src/usr/local/pkg/miniupnpd.xml

```diff
@@ -140,9 +140,7 @@
 				<![CDATA[
 				STUN Server, either hostname or IP address.</br></br>
 
-				Some public STUN servers:</br>
-				stun.xten.com</br>
-				stun.l.google.com (on non standard port 19302)</br>
+				Some public STUN servers: stun.cloudflare.com and stun.counterpath.com</br>
```

Comment:
Personally I'd prefer to leave this to the online docs rather than including it in the GUI. Since it's already there however, keep the same format / line breaks.